src/metabase/automagic_dashboards/filters.clj

```diff
@@ -107,46 +98,34 @@
                     (some-> fingerprint :global :distinct-count (< 2)))))
 
 (defn add-filters
-  "Add up to `max-filters` filters to dashboard `dashboard`. Takes an optional
-   argument `dimensions` which is a list of fields for which to create filters, else
-   it tries to infer by which fields it would be useful to filter."
-  ([dashboard max-filters]
```

Comment:
The most significant change is deleting this arity and the now-unused `candidates-for-filtering` above.